broker/broker.go

```diff
@@ -306,7 +313,7 @@ func (b *S3Broker) Bind(
 			bucketDetails, err := b.bucket.Describe(bucketName, b.awsPartition)
 			if err != nil {
 				if err == awss3.ErrBucketDoesNotExist {
-					errc <- brokerapi.ErrInstanceDoesNotExist
+					errc <- apiresponses.ErrInstanceDoesNotExist
```

Comment:
is apiresponse supposed to be domain?

Reply:
Nope, the errors are imported from a different package. I was getting errors in VSCode when trying to import from `domain`